Alexandra/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>Tea shop</title>
+</head>
+
+<body>
+    <div class="container">
+        <nav id="topBar">
+            <b> Trending deals </b> &nbsp;&nbsp; <a class="topbar" href="https://joesteacompany.com/">See all Deals</a>
+        </nav>
+        <div class="top-container">
+            <div class="tea row">

```

Comment:
tea is too generic. also for the row you use `tea row` but for the column you use `teacolumn`. Inconsistency. Try, tea-card or something more descriptive :D